docs/_book.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+upper_tabs:
+  # Tabs left of dropdown menu
+  - include: /_upper_tabs_left.yaml
+  - include: /api_docs/_upper_tabs_api.yaml
+  # Dropdown menu
+  - name: Resources
+    path: /resources
+    is_default: true
+    menu:
+      - include: /resources/_menu_toc.yaml
+    lower_tabs:
+      # Subsite tabs
+      other:
+        - name: Examples
+          contents:
+            - title: Triplet Loss
+              path: /addons/examples/losses_triplet
+            - title: Image Ops
+              path: /addons/examples/image_ops
+            - title: Normalization Layers
+                path: /addons/examples/layers_normalizations
+            - title: Weight Normalization Layer
+                path: /addons/examples/layers_weightnormalization
+            - title: LazyAdam Optimizer
+                path: /addons/examples/optimizers_lazyadam
+        - name: API
+          skip_translation: true
+          contents:
+            - include: /addons/api_docs/python/_toc.yaml
```

Comment:
@facaiy @tensorflow/sig-addons-maintainers How do we feel about keeping the latest releases API docs on the master branch?

Reply:
Sean, do you mean template files (say _book.yaml etc), or API documents (generated by doc tools)? My initial thought, template files make sense, while API documents not. What do you think?

Reply:
So template files yes. Generated API docs I'm okay with leaving off master seeing as we'll have a new API link in the README.